fe1-web/src/core/network/JsonRpcApi.ts

```diff
@@ -83,6 +84,28 @@ export async function subscribe(
   // propagate the catch() with the full error message, as it needs to be handled on a higher level
 }
 
+/**
+ * Unsubscribe from a channel
+ *
+ * @param channel - The channel from which we want to unsubscribe
+ * @param connections - An optional list of network connection if the message should only be sent on a subset of connections
+ */
+export async function unsubscribe(
+  channel: Channel,
+  connections?: NetworkConnection[],
+): Promise<void> {
+  const request = new JsonRpcRequest({
+    method: JsonRpcMethod.UNSUBSCRIBE,
+    params: new Unsubscribe({
+      channel,
+    }),
+    id: AUTO_ASSIGN_ID,
+  });
+
+  await getNetworkManager().sendPayload(request, connections);
+  // propagate the catch() with the full error message, as it needs to be handled on a higher level
```

Comment:
it might be worthwhile to update the comment, given the `async/await` pattern usage.
```suggestion
  // do not catch exceptions at this level, as the full error message needs to be handled at a higher level
```